core/common/boot/src/test/java/org/eclipse/edc/boot/system/ExtensionLoaderTest.java

```diff
@@ -98,6 +100,33 @@ void loadMonitor_whenNoMonitorExtension() {
         assertTrue(monitor instanceof ConsoleMonitor);
     }
 
+    @ParameterizedTest
+    @EnumSource
+    void loadMonitor_programArgsSetConsoleMonitorLogLevel(ConsoleMonitor.Level level) {
+
+        var monitor = ExtensionLoader.loadMonitor(new ArrayList<>(), ConsoleMonitor.LEVEL_PROG_ARG + "=" + level.toString());
```

Comment:
I would also test various combinations of spaces, upper, lower case etc.:

`"--log-level = INFO"`, `"--log-level= INFO"`, `"--log-level =INFO"`, `"--log-level=info"`,...

Not saying all of those should get accepted, IMO it's fine to only accept `"--log-level=XYZ"`, but to have a definitive test and to guard against regression.